Automattic/wp-calypso
# Site Logs: Persist site log type between site selection

Fixes https://github.com/Automattic/dotcom-forge/issues/7543

> When we're viewing web server logs in a site, and then switch to another site, we should still be seeing web server logs, not getting back to PHP logs.

## Proposed Changes

* Adds feature for both the PHP and Web site logs to allow the site log type (php/web) to persist when a user changes the site selection
* Uses redux to save the `logType` state so that when we move between sites, the logType will persist.
* The is no redirect on `/site-logs/:site` - it will work from that URL now by either defaulting to rendering 'php' logs or using the logType in state.
* This removes the clearing of filters, not sure if we should do this but it made sense when testing.

## Why are these changes being made?


* Improve the UX

## Testing Instructions



* Go to `/sites` and click on an atomic site and go to the `/site-log/{selected-site}/web` for that site - then select another atomic site (where site logs feature should be supported) and the site management panel should load the `web` site logs for the selected site.

## Pre-merge Checklist
